setup.py

```diff
@@ -2,14 +2,9 @@
 import sys
 import numpy
 
+from Cython.Build import cythonize
 from setuptools import Command, Extension, setup
 
-# https://github.com/Unidata/cftime/issues/34
```

Comment:
note: direct invocations of `setup.py` as a scripts are deprecated in setuptools, so gh-34 should not be an issue now.